src/st_stuff.c

```diff
@@ -1479,35 +1479,35 @@ void ST_ResetPalette(void)
 void ST_BindSTSVariables(void)
 {
   M_BindBool("sts_colored_numbers", &sts_colored_numbers, NULL,
-             false, ss_stat, wad_yes, "1 to enable use of color on status bar");
+             false, ss_stat, wad_yes, "Colored numbers on the status bar");
   M_BindBool("sts_pct_always_gray", &sts_pct_always_gray, NULL,
              false, ss_stat, wad_yes,
-             "1 to make percent signs on status bar always gray");
+             "Percentage signs on the status bar are always gray");
```

Comment:
`Percent signs on the status bar are always gray`

Reply:
Did a little research on _percent_ vs. _percentage_, and it looks like I have to change most -- if not all -- instances of the latter to the former.